cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-aks-sample/src/main/java/spring/cloud/azure/starter/data/cosmos/sample/aks/CosmosdbSpringApplication.java

```diff
@@ -0,0 +1,12 @@
+package spring.cloud.azure.starter.data.cosmos.sample.aks;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CosmosdbSpringApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(CosmosdbSpringApplication.class, args);
```

Comment:
Use space instead of tab as intent.

Reply:
did you mean remove unecessary line? Please confirm